### ISSUE TITLE
Another user timeline

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#24]
```
After using Pino for some time I realize that I really miss this feature
from Gwibber before it started to suck. 

Clicking on a username should open some control (a tab? a separate window?)
with:

- user icon
- a link to his/her Twitter page (may be under the icon itself)
- a link to his/her web site from the profile (a very useful thing)
- a timeline for this user, of course
```

Original issue reported on code.google.com by `isagal...@gmail.com` on 27 Dec 2009 at 1:32

**Comment by GoogleCodeExporter** [comment]
```
Will be done
```

Original comment by `tro...@gmail.com` on 27 Dec 2009 at 2:15
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 27 Dec 2009 at 2:15
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
I don't know if this would be related or should be a new submission, but a My
Timeline option similar to this?
```

Original comment by `FoofyFoo...@gmail.com` on 30 Dec 2009 at 3:08

**Comment by GoogleCodeExporter** [comment]
```
Closed by 81ddd2c44b revision. (Currently without a link to the twitter page).
```

Original comment by `tro...@gmail.com` on 4 Mar 2010 at 2:38
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Ah!!! Can't wait for the package :-)
```

Original comment by `isagal...@gmail.com` on 4 Mar 2010 at 2:46
